application/views/lotw/satupdate.php

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+<h2><?php echo $page_title; ?></h2>
+
+<?php
+print '<table class="table table-striped table-hover">';
```

Comment:
Minor thing:
why print where we use echo all times?

afaik print returns a boolean if printed while echo doesn't. But can't see a logic where the result is needed...